streampark-common/src/main/java/org/apache/streampark/common/enums/ClusterState.java

```diff
@@ -32,7 +32,7 @@ public enum ClusterState implements Serializable {
   /** cluster unknown */
   UNKNOWN(4),
   STARTING(5),
-  CANCELING(6),
+  CANCELLING(6),
```

Comment:
These values are too arbitrary. We can set the status according to certain rules, such as transitioning from "create" to "start" to "stop".  e.g:
```  
CREATED(0),
  STARTING(1),
  RUNNING(2),
  CANCELLING(3),
  CANCELED(4),
  FAILED(5),
  KILLED(6);
  LOST(7),
  UNKNOWN(8)
```

Reply:
Due to historical status reasons, directly modifying the enumeration value may cause confusion in the historical status of the `flink cluster`. So I kept the original enumeration values.

Reply:
> Due to historical status reasons, directly modifying the enumeration value may cause confusion in the historical status of the `flink cluster`. So I kept the original enumeration values.

thanks for the clarification